freqtrade/freqai/torch/PyTorchModelTrainer.py

```diff
@@ -40,23 +39,27 @@ def __init__(
             state_dict and model_meta_data saved by self.save() method.
         :param model_meta_data: Additional metadata about the model (optional).
         :param data_convertor: convertor from pd.DataFrame to torch.tensor.
-        :param max_iters: The number of training iterations to run.
-            iteration here refers to the number of times we call
-            self.optimizer.step(). used to calculate n_epochs.
+        :param max_iters: used to calculate n_epochs. The number of training iterations to run.
+            iteration here refers to the number of times optimizer.step() is called.
+            ignored if n_epochs is set.
+        :param n_epochs: The maximum number batches to use for evaluation.
         :param batch_size: The size of the batches to use during training.
-        :param max_n_eval_batches: The maximum number batches to use for evaluation.
         """
         self.model = model
         self.optimizer = optimizer
         self.criterion = criterion
         self.model_meta_data = model_meta_data
         self.device = device
         self.max_iters: int = kwargs.get("max_iters", 100)
+        self.n_epochs: Optional[int] = kwargs.get("n_epochs", None)
```

Comment:
I am glad that we are switching to let users define epochs directly from the config. I felt that the original way was a bit hard for users to control/understand.

I would suggest we actually use `n_epochs` by default instead of `max_iters`. So I would swap `kwargs.get("max_iters", None)` and `kwargs.get("n_epochs", 10)`.


 Can we also add this to the parameter list documentation so that users know they can access this?

Reply:
I'll add `n_epochs` to the docs.

I am considering removing the defaults altogether since they were never intended to function as "good" default values. What are your thoughts on this?

Reply:
You can remove them if you like. In this case we would want to catch the values if they are undefined and yield a descriptive error.